### ISSUE TITLE
My Friend Cant Connect To Wynncraft Servers

### ISSUE THREAD

**Issue by Ken-Kaneki** [Wynncraft/Issues#8155]
When he click on one of the servers to join he gets to where it says create your character and then it disconnects him he's deleted it and re-downloaded it and tried it again and again but it won't let him get any further then the create your character part.

**Comment by Ken-Kaneki** [comment]
.

**Comment by LordHelmchen98** [comment]
Is this still happening and if so can you describe it more precisely? What exactly happens? Does he get error messages when he tries to join with a class?

**Comment by Ken-Kaneki** [comment]
Yes, it is still happening  when he joins it showed the class screen for like 10 sec and then it disconnects him it doesn't show an error or anything like that it just says disconnected

**Comment by LordHelmchen98** [comment]
I'm not really sure why this happens, but I have two guesses. It could be because of the connection to the servers or because of the resource pack. If it is because of the connection I have no idea what could help, but if it is because of the resource pack he could try to delete the packs in his server resource pack folder.
